libraries/core/src/config.rs

```diff
@@ -236,15 +243,32 @@ pub fn format_duration(interval: Duration) -> FormattedDuration {
     FormattedDuration(interval)
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, JsonSchema)]
 pub struct NodeRunConfig {
+    /// Inputs for the nodes as a map from input ID to input configuration
```

Comment:
```suggestion
    /// Inputs for the nodes as a map from input ID to `<node_id>/<output_id>`.
```

Reply:
So I have replaced the documentation. I have just removed the \< \> so that documentation generation does not take those for html tags.